packages/@uppy/transloadit/src/Client.js

```diff
@@ -1,4 +1,22 @@
 const fetchWithNetworkError = require('@uppy/utils/lib/fetchWithNetworkError')
+const NetworkError = require('@uppy/utils/lib/NetworkError')
+
+function fetchJSON (...args) {
+  return fetchWithNetworkError(...args).then(response => {
+    if (response.status === 429) {
+      // If the server asks the client to rate limit, reschedule the request 5s later.
+      return new Promise((resolve, reject) => {
+        setTimeout(() => fetchJSON(...args).then(resolve, reject), 5_000)
```

Comment:
not too happy with multiple rate-limit-like implementations laying around. Comes down the tribal knowledge where we are rate limiting and with what implementation. Can we reuse logic?

Reply:
We would need a global request queue (maybe in `@uppy/core`) that would handle all of this.

Reply:
Alright let's think about that another time. Can you add a TODO comment and change the reschedule from 5 seconds to 1 or 2?